src/event/types.rs

```diff
@@ -1,4 +1,5 @@
 use chrono::{DateTime, Utc};
+use std::prelude::v1::*;
```

Comment:
Can we enable these imports around the sdk only if the `no_std` feature is enabled?